python_modules/libraries/dagster-dbt/dagster_dbt/dagster_dbt_translator.py

```diff
@@ -520,6 +521,42 @@ def get_automation_condition(self, dbt_resource_props: Mapping[str, Any]) -> Opt
             auto_materialize_policy.to_automation_condition() if auto_materialize_policy else None
         )
 
+    @public
```

Comment:
Can we remove the public decorator? This should be kept for internal usage for now